src/styles/index.scss

```diff
@@ -1,3 +1,56 @@
+@import '/src/styles/variables';
+
 body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
```

Comment:
It's better to separate general styles to a different file. This way, you can keep your styles organized and it's easier to manage them.